src/main/scala/uk/gov/hmrc/play/http/HeaderCarrier.scala

```diff
@@ -114,10 +120,25 @@ object HeaderCarrier {
       headers.get(HeaderNames.trueClientPort),
       headers.get(HeaderNames.googleAnalyticTokenId),
       headers.get(HeaderNames.googleAnalyticUserId),
-      getDeviceId(cookies, headers)
+      getDeviceId(cookies, headers),
+      headers.get(HeaderNames.akamaiReputation).map(AkamaiReputation),
```

Comment:
Do we want to hard code the akamaiReputation header or will this now be covered by the blacklist changes?